src/edu/washington/escience/myria/coordinator/catalog/MasterCatalog.java

```diff
@@ -889,7 +889,7 @@ private Schema getSchema(@Nonnull final SQLiteConnection sqliteConnection, @Nonn
     try {
       SQLiteStatement statement =
           sqliteConnection
-              .prepare("SELECT col_name,col_type FROM relation_schema WHERE user_name=? AND program_name=? AND relation_name=?; ORDER BY col_index ASC");
+              .prepare("SELECT col_name,col_type FROM relation_schema JOIN relations USING (user_name,program_name,relation_name) WHERE user_name=? AND program_name=? AND relation_name=? ORDER BY col_index ASC");
```

Comment:
My fault. I initially had the ```tombstone``` filter but I took it out because my delete call calls getSchema() at some point.  I forgot to take out the ``JOIN relations```